src/graphql/ProductFragment.graphql

```diff
@@ -47,7 +47,17 @@ fragment ProductFragment on Product {
   }
   metafields(identifiers: [
     {namespace: "taiga", key: "preOrderStopCondition"},
-    {namespace: "taiga", key: "preOrderDateEstimate"}
+    {namespace: "taiga", key: "preOrderDateEstimate"},
+    {namespace: "productListing", key: "preOrderStopCondition"},
+    {namespace: "productListing", key: "launchType"},
+    {namespace: "productListing", key: "startingQuantity"},
+    {namespace: "productListing", key: "endingQuantity"},
+    {namespace: "productListing", key: "headline"},
+    {namespace: "productListing", key: "sizing"},
+    {namespace: "productListing", key: "campaignStart"},
+    {namespace: "productListing", key: "campaignEnd"},
```

Comment:
I see in the document that there is a `campaignDuration` metafield defined instead of this `campaignEnd` one - was that changed offline?

Reply:
Ah, yeah it should be `campaignDuration`, there was a misalignment between the field name and what should be here.